### PR TITLE
Daiquiri is set up for logging

### PR DESCRIPTION
**Pull request by rakaar**
1) included to run `pre-commit install` in `README.md`
2) The code for logging using daiquiri is in `daiquiri/__init__.py`

### CODE REVIEW

File: daiquiri/__init__.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import logging
```
Comment (thealphadollar) [inline daiquiri/__init__.py]: This is the issue; you have named a module `daiquiri`, so when you are importing it, it is importing the local one and not the installed module. Please avoid name clashes like this in future.

What you should do is shift this code to `/app/main/logging_config.py`. Also import the logging from daiquiri into Flask app. You need to experiment a bit on this about how to get it working with native flask logging.

Comment (rakaar) [reply daiquiri/__init__.py]: > Also import the logging from daiquiri into Flask app. You need to experiment a bit on this about how to get it working with native flask logging.
I didn't understand exactly?

**Comment by thealphadollar** [comment]
We will be logging using `app.logger` and this needs to be setup.

We need to find a way to setup config using daiquiri and then have the same
config with flask. If we cannot do this, there is no point of daiquiri.

**Comment by thealphadollar** [comment]
Not required anymore.

**Pull request closed without merging**